src/test/java/com/puppycrawl/tools/checkstyle/internal/utils/XmlUtil.java

```diff
@@ -81,6 +81,7 @@ public static Set<Node> getChildrenElements(Node node) {
             }
         }
 
+
```

Comment:
please revert this extra empty line

Reply:
sometimes IDEA lags and this kind of edits happen.fixed now. 
which IDE do you use though ?

Reply:
Idea.
But I always check diff in terminal to spot unexpected changes.